### PR TITLE
Display level up message for #6

### PR DESCRIPTION
**Pull request by parrajustin**
- added a getLevel function inside player
- added level variable within game to keep track of level
- added congratulation message on intermission screen

### CODE REVIEW

File: Turn/src/Game.cpp [hunk 1]
```diff
@@ -203,7 +203,14 @@ void Game::Intermission(){
     for (int choice=0; IsPlaying;){
         ClearScreen();
         cout << "*--------- Intermission ----------* " << endl << endl;
-		_Player->DisplayInventory();
+		
+	int tempLevel = _Player->GetLevel();
+	if (_Level != tempLevel) {
+		_Level = tempLevel;
+		cout << "Congratulation on leveling up!" << endl;
+	}
```
Comment (tagniam) [inline Turn/src/Game.cpp]: Hmm, on second thought-- this level up message would look better placed after winning a battle (after showing the items you won), if you're up for it.

Comment (parrajustin) [reply Turn/src/Game.cpp]: I believe it already does this inside Player::AddExperience

**Comment by tagniam** [comment]
Thanks @parrajustin!

**Comment by parrajustin** [comment]
image

Now it shows the level up message after seeing the items you won.

**Comment by parrajustin** [comment]
Thanks for letting me help @tagniam!

**Pull request merged**